kind/docs/todo.md

```diff
@@ -24,6 +24,7 @@ A non-exhaustive list of tasks (in no-particular order) includes:
 - [ ] support multiple overlay networks
 - [x] support advanced configuration via config file
   - [x] kubeadm config template override
+  - [ ] node lifecycle hooks
```

Comment:
Added a TODO to follow up and implement these. should be relatively straight forward but this PR is already pretty big.
I think we should also possibly expose some env to these when we do.